flasc/utilities.py

```diff
@@ -13,7 +13,7 @@
 
 import datetime
 
-import numba

```

Comment:
Is it necessary to maintain this as a commented code or ok to delete? It seems that numba will not be used so, in my opinion, leaving these lines in but commented means its out of context.

Reply:
`numba` has been a big source of dependency errors and code inconsistency. However, when it did work, it sped up certain time resampling operations a lot. I'm not currently looking at any high-resolution data, but would be worth reconsidering this option when I do.